sopel/cli/run.py

```diff
@@ -533,18 +520,22 @@ def command_legacy(opts):
         print_version()
         return
 
+    # TODO: allow to use a different homedir
```

Comment:
Just a question: do you mean that you should be able to specify another directory to search for config files from the command line? 

If yes, nothing further :smile: 

If not, then I think this needs some clarification/discussion as to what is actually planned. I just wanted to confirm it's nothing will end up breaking without this `TODO`. The few runs I did to test various interpretations of this message seemed OK so far, so I'm not too worried :ok_hand:

Anyway, this is related to the CLI/wizard, and not the plugin system, so it doesn't preclude approval :+1:  I put it here since it wasn't present until now (that I can tell).

Reply:
Yeah, we discussed on IRC that we should probably have a ``--homedir`` option to define the homedir at runtime, so you can do that:

    sopel start --homedir=/etc/sopel -c freenode

in order to load the configuration file `/etc/sopel/freenode.cfg` or to do that:

    sopel-config list --homedir=/etc/sopel

to list configuration files available in `/etc/sopel`. The problem is visible 2 lines below when we use `utils.find_config(config.DEFAULT_HOMEDIR, ...)`, and also with the config wizard.

Reply:
Oh good. I did see the `utils.find_config(...` and was able to piece it together, but was initially  worried the `TODO` was related to the `core.homedir` setting. I quickly figured out that it was not actually the same, but it brings me to the question of whether `homedir` is the right term for that specific reference (and even `core.homedir` seems a little misnomer-y). Anyway, that can be dealt with in the appropriate context  when the time comes :grin: